### ISSUE TITLE
Toggle is not working on Android 4.2.2

### ISSUE THREAD

**Issue by fangyidong** [twbs/ratchet#766]
It seems that Android 4.2.2 doesn't have a CustomEvent object. The workaround is to create the CustomEvent in the following way:

```javascript
    var theEvent= document.createEvent("CustomEvent");
    theEvent.initCustomEvent('toggle', true, true, {isActive: slideOn, source: toggle})

    try{
        document.dispatchEvent(theEvent);// dispatchEvent of child DOM elements doesn't seem to work on Android 4.2.2
    }
    catch(err){
        alert("err: " + err);
    }

```
And you have to listen to the toggle event at Document level.

**Comment by Johann-S** [comment]
It's fixed it will be present in the next release of Ratchet

**Comment by cvrebert** [comment]
Already fixed by #748.
